dropwizard-logging/src/main/java/io/dropwizard/logging/DefaultLoggingFactory.java

```diff
@@ -170,10 +192,24 @@ private Logger configureLevels() {
 
         root.setLevel(level);
 
-        for (Map.Entry<String, Level> entry : loggers.entrySet()) {
-            loggerContext.getLogger(entry.getKey()).setLevel(entry.getValue());
-        }
+        if (advancedLoggers != null && advancedLoggers.size() > 0) {
```

Comment:
@nickbabcock it was what I thought needed, it can be changed easily. Thanks for you feedback :+1: